server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -336,6 +336,12 @@ public class ShuffleServerConf extends RssBaseConf {
       .withDescription("Threshold when flushing shuffle data to persistent storage, recommend value would be 256K, "
           + "512K, or even 1M");
 
+  public static final ConfigOption<String> STORAGE_TYPE_PROVIDER_ENV_KEY = ConfigOptions
```

Comment:
Is this an internal config entry? If yes, could you mark it, rename from  `STORAGE_TYPE_PROVIDER_ENV_KEY` to `__INTERNAL_STORAGE_TYPE_PROVIDER_ENV_KEY` ? WDYT